plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -126,13 +126,20 @@ SecRule REQUEST_METHOD "@pm PROPFIND PUT" \
         ctl:ruleRemoveById=921110"
 
 # Allow the data type 'text/vcard'
+# Allow characters like /../ in files.
```

Comment:
Path traversal is a common security issue. Can you explain why this is necessary? I know you didn't put that exclusion here, but it would be helpful, if you know the reason.

```suggestion
# Allow path traversal characters like /../ in file paths.
```

Reply:
I'm not sure the reasoning behind it and I don't want to break things for other users by overtightening rules. I'm on libmodsecurity 3 and iirc excluding by rule id ranges doesn't work so maybe it's safe to remove?

Reply:
If you don't know, than that's fine. Leave it as it is.

Reply:
I'll leave as is then, I'll fix it in the future if I do know.